challenge.rb

```diff
@@ -1,26 +1,26 @@
 
 def capitalize_each_string(input)
-  #implement your solution here
+  input.map { | animal | animal.capitalize }
```

Comment:
looks good. are you familiar with ruby's to_proc method? its super common and would help you shorten this a bit.

https://stackoverflow.com/questions/14881125/what-does-to-proc-method-mean

Reply:
@jaybobo Ahh gotcha. So something more like this `input.map(&:capitalize)` ?